devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -46,6 +46,11 @@ metadata {
 
 		// Salus
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "SALUS", model: "SX885ZB", deviceJoinName: "Salus miniSmartplug"
+        
+        //AduroSmart
+        fingerprint profileId: "0104", deviceId: "0051", inClusters: "0000, 0003, 0004, 0005, 0006, 0B04, 1000, 0702", outClusters: "0019", manufacturer: "AduroSmart Eria", model: "AD-SmartPlug3001", deviceJoinName: "Eria Zigbee Smart Plug"
```

Comment:
@adurosmart Could you change indentation to tabs instead of spaces?

Reply:
I already change indentation to tabs.
Thank you to review.